models/webhooks_entities_webhook_entity.go

```diff
@@ -14,10 +14,10 @@ import (
 	"github.com/go-openapi/validate"
 )
 
-// WebhookEntity WebhookEntity model
+// WebhooksEntitiesWebhookEntity Webhooks_Entities_WebhookEntity model
 //
-// swagger:model WebhookEntity
-type WebhookEntity struct {
+// swagger:model Webhooks_Entities_WebhookEntity
+type WebhooksEntitiesWebhookEntity struct {
```

Comment:
1. lol @ that name
2. is this intentional change? may need to do a batch find and replace on any existing library that imports it, but as long as we're not flip-flopping back and forth between the two should be OK

Reply:
Well that's unfortunate… I don't think it'll change again, so I'm not too worried about flip-flopping